requirements.txt

```diff
@@ -2,7 +2,7 @@ blobfile
 fire
 httpx
 huggingface-hub
-llama-models>=0.0.24
+llama-models==0.0.24
```

Comment:
I think this should be >=0.0.36 and cannot be changed. What issue does this cause?

Reply:
It was causing the naming of the models in inference adapter. This is feed now.